### ISSUE TITLE
execute_script throwing "SystemStackError"

### ISSUE THREAD

**Issue by westlakem** [watir/watir#684]
## Meta -
Watir Version: 6.8.4
Selenium Version: selenium-webdriver 3.7.0
Browser Version: Chrome 62
Browser Driver Version: 2.33
OS Version: Win 10

## Expected Behavior -
`browser.execute_script('return performance.getEntriesByType("resource")')` to return the resource entries from the browser.
## Actual Behavior -
`browser.execute_script('return performance.getEntriesByType("resource")')` errors with 
 `SystemStackError: stack level too deep`

## Steps to reproduce -
```
 def self.take_measurement
    lambda do |browser|
      current_resource_load = browser.execute_script('return performance.getEntriesByType("resource")')  #this is the line it errors on
      current_resource_load.each do |downloaded_resource|
        performance_metrics = parse_performance_details(browser, downloaded_resource)
        @browser_performance_metrics.add(performance_metrics)
      end
    end
  end

browser = Watir::Browser.new :chrome
browser.after_hook.add(take_measurement)
```
navigate to a webpage and try clicking around on elements (in your script)

**Comment by titusfortner** [comment]
Ah, can you try this with Watir 6.7.3? I changed something in 6.8.0 that might have caused this.

**Comment by titusfortner** [comment]
Pretty sure this will fix it: https://github.com/watir/watir/commit/e0df7adf2a064783f4fe3f9ab3e3b6eb0c9f02f8
I'll get it released soon.

**Comment by westlakem** [comment]
@titusfortner 
I'm pretty sure your fix didn't get put into the 6.9 release version
![image](https://user-images.githubusercontent.com/4505074/33025727-ba7e9e44-cddc-11e7-8d29-450d62b00cae.png)

**Comment by titusfortner** [comment]
Ah, no it didn't. Was going to make a point release on Saturday and got caught up in other things. I'll get it out in a bit.

**Comment by titusfortner** [comment]
Released in 6.9.1

**Comment by westlakem** [comment]
Looks good! thanks!
